lib/block-patterns.php

```diff
@@ -70,3 +70,85 @@
 						<!-- /wp:query -->',
 	)
 );
+
+register_block_pattern(
+	'query/posts-grid',
+	array(
+		'title'      => __( 'Post Grid', 'gutenberg' ),
+		'blockTypes' => array( 'core/query' ),
+		'categories' => array( 'Query' ),
+		'content'    => '<!-- wp:query {"queryId":11,"query":{"perPage":12,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"layout":{"type":"flex","columns":3}} -->
```

Comment:
I just glanced it (will look at it better tomorrow) but `queryId` should not be provided and `perPage` affects the preview, so I would suggest to set it at the minimum to serve the pattern's purpose. For example if we have a grid with two columns, a `perPage:2` could be enough I think. 

Cool patterns though 💯

Reply:
Thanks! I've removed the `queryId`s, and adjusted the perPage to the minimum that will show a decent preview for each pattern. 👍